core/trino-main/src/main/java/io/trino/sql/planner/planprinter/PlanPrinter.java

```diff
@@ -884,7 +885,12 @@ public Void visitUnnest(UnnestNode node, Void context)
                 name = node.getJoinType().getJoinLabel() + " Unnest";
             }
             else if (!node.getReplicateSymbols().isEmpty()) {
-                name = "CrossJoin Unnest";
+                if (node.getJoinType() == INNER) {
```

Comment:
I do not understand the naming pattern here.

Reply:
It seems like `PlanPrinter` wasn't updated when we added LEFT unnest. It still printed `CrossJoin Unnest`, no matter what the type actually was.
I left the preexisting behavior to print `CrossJoin` for INNER with no filter.
I think it could be simplified to always print the joinLabel + " Unnest".